src/components/ModalPortal.js

```diff
@@ -36,6 +41,13 @@ export default {
   },
 
   methods: {
+    getModals() {
```

Comment:
Let's rename with `visibleModalStack` because `getModals` does not imply what kind of modals are returned.